ratelimitbackend/admin.py

```diff
@@ -1,5 +1,4 @@
-from django.contrib.admin import *  # noqa
```

Comment:
The idea behind `import *` is to be able to use `ratelimitbackend.admin` wherever `django.contrib.admin` is used. E.g. if you do `from ratelimitbackend import admin`, you can still write `admin.ModelAdmin` without having to import from the Django admin under another name.